app/main.py

```diff
@@ -1,3 +1,66 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[float, int]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[float, int, Distance]) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(km=self.km + other)
+
+    def __iadd__(self, other: Union[float, int, Distance]) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: Union[float, int]) -> Distance:
+        return Distance(km=self.km * other)
+
+    def __truediv__(self, other: Union[float, int]) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Union[float, int, Distance]) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif self.km < other:
```

Comment:
you can write only this because `self.km < other` returns `True` or `False` like in above code
```suggestion
        return self.km < other
```

Reply:
change here and all of the code